src/data/objects/ipaddress.yaml

```diff
@@ -31,11 +31,14 @@ schema:
     type: String
     value: 255.255.255.0
     description: The subnet mask.
+  public:
+    type: Boolean
+    value: true
+    description: Whether this is a public of private IP address.
   type:
-    type: Enum
-    subtype: IPAddressType
-    value: public
-    description: The type of IP Address, either public or private.
+    type: String
+    description: The type of IP Address.
+    value: ipv4
 enums:
```

Comment:
This enum is now irrelevant - I'd change is as described above, but it could also be removed